libs/report/committers/src/lib/parse-monthly.ts

```diff
@@ -48,3 +33,20 @@ export function parseMonthly(
   }
   return monthly;
 }
+
+function monthlyDataHumanReadable(data: MonthlyData) {
+  return {
+    ...data,
+    start: format(data.start, 'yyyy-MM-dd'),
+    end: format(data.end, 'yyyy-MM-dd'),
+  };
+}
+
+export function outputMonthlyCommitters(commits: MonthlyData[]) {
+  const mapped = commits.map((c) => monthlyDataHumanReadable(c));
+  console.table(mapped);
```

Comment:
The committers object will not look great as a table cell value. Should we format it separately or maybe just display the total number of commits per month?

Reply:
Sounds good to me :+1: